app/src/main/java/com/tvtracker/Media.kt

```diff
@@ -12,11 +12,13 @@ class Media : GestureDetector.SimpleOnGestureListener() {
         return super.onDown(e)
     }
 
-    override fun onFling(e1: MotionEvent?, e2: MotionEvent?, velocityX: Float, velocityY: Float): Boolean {
+    override fun onFling(e1: MotionEvent?, e2: MotionEvent?, velocityX: Float, velocityY: Float):
```

Comment:
Code has a column limit of 100 characters 
Exceptions Include:
- Lines whereobeying the column limit is not possible (for example, a long URL in KDoc)
- package and import statements
- Command lines in a comment that may be cut-and-pasted into a shell
https://developer.android.com/kotlin/style-guide